cartridge/auth.lua

```diff
@@ -343,6 +344,9 @@ local function login(req)
         }
     elseif err ~= nil then
         log.error('%s', err)
+        if type(err) == 'table' and vars.disable_errstack then
```

Comment:
```suggestion
        if type(err) == 'table' and vars.disable_errstack == true then
```
Otherwise, you may have `box.NULL` in `vars.disable_errstack`